src/commands/projects-list.ts

```diff
@@ -0,0 +1,54 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+// TODO(caot): Replace with proper import
+import Table = require("cli-table");
```

Comment:
@bkendall, using `import * as from "cli-table"` makes the ts compiler complain here, and this is the work around (I saw this pattern in some other command already). Do you understand why it get complained?

Reply:
That's a very long discussion, but basically boils down to how modules and import/require work. This is fine to do.

Reply:
Thanks, :+1:. Cleaning up the TODO